scripts/update-drafts.ts

```diff
@@ -1,3 +1,7 @@
+// Updates files in features/draft/spec/* with BCD keys mentioned in specs
+// `npm run update-drafts` updates all
+// `npm run update-drafts -- [key]` only updates features with a `shortname` that includes `key`.
```

Comment:
Do I understand correctly that this matches a key, not a file name? I think that's surprising and slightly inconvenient (e.g., it'd be nice to be able to use shell globbing or xargs to run this).

What do you think of using file names as arguments, then getting the key from the filename (kinda like [dist does](https://github.com/web-platform-dx/web-features/blob/3af1c7de6adc6a3227c574986a6f99335037b279/scripts/dist.ts#L323) for yml and dist files)?

Or if you like the idea of providing a partial string (like your `ext` example), maybe we _do_ need yargs, so we can have something like `--by-key` and `--by-path` options.

Reply:
Yes, you're correct that it takes a key, so `paint-order` would match but `paint-order.yml` or `features/draft/spec/css-transforms-2.yml` would not. 

I agree that file paths would make sense to support, I can update to do that (and will go ahead and add yargs).